presto-hive/src/main/java/com/facebook/presto/hive/HiveColumnHandle.java

```diff
@@ -52,6 +53,11 @@ public class HiveColumnHandle
     public static final HiveType BUCKET_HIVE_TYPE = HIVE_INT;
     public static final TypeSignature BUCKET_TYPE_SIGNATURE = BUCKET_HIVE_TYPE.getTypeSignature();
 
+    public static final int ROW_ID_COLUMN_INDEX = -16;
```

Comment:
per https://github.com/prestodb/presto/pull/22008#discussion_r1553739252, change this to be a value between -1 and -12

Reply:
Nice detective work. Done.